src/tools/roc_recv/main.cpp

```diff
@@ -64,55 +64,14 @@ int main(int argc, char** argv) {
         break;
     }
 
-    node::ContextConfig context_config;
-
-    if (args.packet_limit_given) {
-        if (args.packet_limit_arg <= 0) {
-            roc_log(LogError, "invalid --packet-limit: should be > 0");
-            return 1;
-        }
-        context_config.max_packet_size = (size_t)args.packet_limit_arg;
-    }
-
-    if (args.frame_limit_given) {
-        if (args.frame_limit_arg <= 0) {
-            roc_log(LogError, "invalid --frame-limit: should be > 0");
-            return 1;
-        }
-        context_config.max_frame_size = (size_t)args.frame_limit_arg;
-    }
-
-    core::HeapArena heap_arena;
-
-    node::Context context(context_config, heap_arena);
-    if (!context.is_valid()) {
-        roc_log(LogError, "can't initialize node context");
-        return 1;
-    }
-
-    sndio::BackendDispatcher backend_dispatcher(context.arena());
-
-    if (args.list_supported_given) {
-        if (!sndio::print_supported(backend_dispatcher, context.arena())) {
-            return 1;
-        }
-
-        if (!address::print_supported(address::ProtocolMap::instance(),
-                                      context.arena())) {
-            return 1;
-        }
-
-        return 0;
-    }
-
     pipeline::ReceiverConfig receiver_config;
 
     sndio::Config io_config;
     io_config.sample_spec.set_channel_set(
         receiver_config.common.output_sample_spec.channel_set());
 
-    if (args.frame_length_given) {
-        if (!core::parse_duration(args.frame_length_arg, io_config.frame_length)) {
+    if (args.frame_len_given) {
+        if (!core::parse_duration(args.frame_len_arg, io_config.frame_length)) {
             roc_log(LogError, "invalid --frame-length: bad format");
```

Comment:
Here and in other places, `--frame-length` should be replaced with `--frame-len`.